tests/e2e-tests/config/default.json

```diff
@@ -1,15 +1,62 @@
 {
-	"url": "http://example.com",
-	"users": {
-		"admin": {
-			"username": "",
-			"password": ""
-		},
-		"customer": {
-			"username": "",
-			"password": ""
-		}
-	},
-	"startBrowserTimeoutMs": 30000,
-	"mochaTimeoutMs": 120000
+  "url": "http://localhost:8084/",
```

Comment:
Possible to pass this and credentials below from the environment variable? That way we can run tests on any URL.

Reply:
I am not sure immediately if this is possible. We are setting the URL as well as the admin and customer user details in the `docker-compose.yaml`. When setting the URL we also need to identify the port for localhost. Could be that I am missing something simple? It will be nice to run tests on any URL. This is also something that I can work on in the next stage of this project. What do you think?

Reply:
Sure, let's punt this to next stage.

Reply:
I created an issue to document the task: https://github.com/woocommerce/woocommerce/issues/25275